cohort-engine/src/main/java/com/ibm/cohort/engine/CqlEngineWrapper.java

```diff
@@ -179,11 +179,11 @@ public IGenericClient getTerminologyServerClient() {
 	 *                       specified <code>expressions</code> will be executed. At
 	 *                       least one value is required.
 	 * @param callback       callback function to be evaluated once per context per
-	 *                       executed define (required).
+	 *                       executed define (required).             
 	 */
 	protected void evaluateExpressionByExpression(final String libraryName, final String libraryVersion,
```

Comment:
There didn't seem to be any catchable exceptions in the class so I removed the throws.